_pages/cfp.md

```diff
@@ -1,10 +1,55 @@
 ---
 layout: page
 title: "BSidesSF 2022 Call For Participation"
-redirect_from: "/cfw"
-redirect_from: "/cfw.html"
----
+permalink: /cfp.html
+--- 
+
+**What:** BSidesSF 2022 Call For Participation for Talks and Workshops
+
+BSidesSF is a non-profit organization designed to advance the body of Information Security knowledge, by providing an annual, open forum for discussion and debate for security practitioners. We produce a conference that is a source of education, collaboration, and continued conversation for information technologists. The technical and academic presentations at BSidesSF are given in the spirit of peer review and advanced knowledge dissemination. This allows the field of Information Security to grow in breadth and depth, and continue in its pursuit of highly advanced, scientifically based knowledge. 
+
+We are a practitioner focused event that attracts a wide range of attendees ranging from engineers & security leaders to executive decision makers. This is a great opportunity for you to showcase your work and share your ideas with industry peers. Bonus points if you can tie your presentation to this year’s conference theme: From the Ground Up! 
```

Comment:
In past years, we have come up with a few examples of talk names that follow the theme. Might be fun to do the same. Or perhaps go in-depth a bit more about the theme and what it means exactly?